R/autoplot.pop_data.R

```diff
@@ -13,14 +13,13 @@
 #' @examples
 #' library(dplyr)
 #' library(ggplot2)
-#' library(ggfortify)
 #'
 #' xs_data <- "https://osf.io/download//n6cp3/" %>%
 #'   load_pop_data() %>%
 #'   clean_pop_data()
 #'
-#' xs_data %>% autoplot(strata = "Country", type = "density")
-#' xs_data %>% autoplot(strata = "Country", type = "age-scatter")
+#' xs_data %>% autoplot(strata <- "Country", type = "density")
+#' xs_data %>% autoplot(strata <- "Country", type = "age-scatter")
```

Comment:
using `<-` for function inputs doesn't work; it's only passing the GitHub checks because you haven't run `document()` since making these changes, and the examples are pulled from the .Rd files.